src/radical/pilot/agent/scheduler/base.py

```diff
@@ -393,42 +395,21 @@ def slot_status(self):
         Returns a multi-line string corresponding to the status of the node list
         '''
 
+        glyphs = {rpc.FREE : '-',
+                  rpc.BUSY : '#',
+                  rpc.DOWN : '!'}
```

Comment:
Additional to `FREE` and `BUSY`, we add `DOWN` to mark cores we cannot use for some reason.  On summit, some cores cannot be addressed via ERF right now.